core/chains/evm/client/send_only_node.go

```diff
@@ -76,40 +87,71 @@ func NewSendOnlyNode(lggr logger.Logger, httpuri url.URL, name string, chainID *
 
 func (s *sendOnlyNode) Start(ctx context.Context) error {
 	return s.StartOnce(s.name, func() error {
-		return s.start(ctx)
+		s.start(ctx)
+		return nil
 	})
 }
 
 // Start setups up and verifies the sendonly node
 // Should only be called once in a node's lifecycle
-// TODO: Failures to dial should put it into a retry loop
-// https://app.shortcut.com/chainlinklabs/story/28182/eth-node-failover-consider-introducing-a-state-for-sendonly-nodes
-func (s *sendOnlyNode) start(startCtx context.Context) error {
+func (s *sendOnlyNode) start(startCtx context.Context) {
+	if s.state != NodeStateUndialed {
+		panic(fmt.Sprintf("cannot dial node with state %v", s.state))
+	}
+
 	s.log.Debugw("evmclient.Client#Dial(...)")
 	if s.dialed {
 		panic("evmclient.Client.Dial(...) should only be called once during the node's lifetime.")
 	}
 
+	// DialHTTP doesn't actually make any external HTTP calls
+	// It can only return error if the URL is malformed. No amount of retries
+	// will change this result.
 	rpc, err := rpc.DialHTTP(s.uri.String())
 	if err != nil {
-		return errors.Wrapf(err, "failed to dial secondary client: %v", s.uri.Redacted())
+		promEVMPoolRPCNodeTransitionsToUnusable.WithLabelValues(s.chainID.String(), s.name).Inc()
+		s.log.Errorw("Dial failed: EVM SendOnly Node is unusable", "err", err)
+		s.setState(NodeStateUnusable)
```

Comment:
Need to return from here.

Reply:
Ooof, well spotted. Will add test case for this